cmd/resource-quota-check/README.md

```diff
@@ -0,0 +1,78 @@
+## Resource Quotas (CPU & Memory)
+
+This check tests if namespace resource quotas `CPU` and `memory` are under a specified threshold or percentage. Namespaces that utilize a lot of `CPU` or `memory` resources can sometimes run into an issue where controllers (_i.e. deployment or replica controllers_) are unable to schedule pods due to insufficient `CPU` or `memory`.
+
+This check lists all namespaces in the cluster and checks if each resource (`CPU` and `memory`) are at an ok percentage.
+
+This check can be configured to use either a `blacklist` or a `whitelist` of namespaces, allowing you to explicitly target or ignore specific namespaces. This can be changed with the environment variables `BLACKLIST` and `WHITELIST` which take `"true" or "false`. This check assumes the use of a `BLACKLIST` on check runs; if both `BLACKLIST` and `WHITELIST` options are enabled, `BLACKLIST` will take precedence. The default lists are:
```

Comment:
What about refactoring this so that both a blacklist or whitelist can be used in any way the user likes?

Here is the matrix of functionality:
- if blacklist is specified, and whitelist is not, then we simply operate on a blacklist
- if blacklist is specified, and whitelist is also specified, then we operate on the whitelist unless the item is in the blacklist
- if blacklist is not specified, but whitelist is, then we operate on a whitelist
- if neither a blacklist or whitelist is specified, then all namespaces are targeted

This may sound pointless, but perhaps someone may use it programmatically in the future and it will be easy for them.  This should also simplify our code, because we can simply ask a few boolean questions to implement all functionality, rather than operating on some mutating state machine that has various types of functionality.

Reply:
Currently only 1 and 3 are implemented:
`if blacklist is specified, and whitelist is not, then we simply operate on a blacklist`
`if blacklist is not specified, but whitelist is, then we operate on a whitelist`

I think here, I would need to add another case to my `switch` statement on `line: 110` in `resources.go`

Reply:
In theory, would it be bad to run two of these checks to achieve the same outcome? 
Have a `resource-quota-blacklist` and `resource-quota-whitelist` check that operate on blacklists and whitelists respectively -- thoughts?